src/components/Dashboard/Dashboard.test.js

```diff
@@ -31,26 +32,40 @@ describe('on initial render', () => {
         const wrapper = shallow(<Dashboard status={initialGameState.status} />);
         expect(wrapper.exists('#round-number')).toEqual(false);
     });
+    it("doesn't display the current word", () => {
+        const wrapper = shallow(<Dashboard currentWord={initialGameState.currentWord} />);
+        expect(wrapper.exists('#message')).toEqual(false);
+    });
+    it("displays the start game button", () => {
+        const wrapper = shallow(<Dashboard status={initialGameState.status}/>);
+        expect(wrapper.exists('button#start-game')).toEqual(true);
+    });
+    describe('on clicking the start game button', () => {
+        it('calls startGame', () => {
+            const spy = jest.spyOn(Dashboard.prototype, 'startGame');
+            const wrapper = shallow(<Dashboard status={initialGameState.status}/>);
+            wrapper.find('button#start-game').simulate('click');
+            expect(spy).toHaveBeenCalled();
+            spy.mockRestore();
+        });
+    });
 });
 
 describe('on start of game', () => {
-    it('displays the current players username', () => {
+    it("displays the current player's username", () => {
         const wrapper = shallow(<Dashboard status={firstRoundGameState.status} currentPlayer={firstRoundGameState.currentPlayer} />);
         expect(wrapper.find('#current-player').text()).toEqual('player1');
     });
     it('displays the correct round number', () => {
         const wrapper = shallow(<Dashboard status={firstRoundGameState.status} roundNum={firstRoundGameState.roundNum}/>);
         expect(wrapper.find('#round-number').text()).toEqual('1');
     });
+    it('displays the correct word', () => {
+        const wrapper = shallow(<Dashboard currentWord={"Freedom"}/>);
+        expect(wrapper.find('#message').text()).toEqual("Word: Freedom");
+    });
     it("doesn't display the start game button", () => {
         const wrapper = shallow(<Dashboard status={firstRoundGameState.status} />);
         expect(wrapper.exists('#start-game')).toEqual(false);
     });
-});
-
-describe('after you have played your turn', () => {
```

Comment:
Why was this test removed?

Reply:
Because the end turn button sits in the PlayWordAndCard component, I thought I should move it in those tests